### PR TITLE
Add guidelines about early merge

### PR DESCRIPTION
**Pull request by rayegun**
@DilumAluthge

### CODE REVIEW

**Comment by DilumAluthge** [comment]
@ericphanson mentioned that there is an existing section that covers some of the same topic, but for a different audience (addressing registry committers as the audience).

Probably we should add links pointing from one section to the other, in both directions?

**Comment by rayegun** [comment]
Where is that section? Just searching for merge I don't see it.

**Review by giordano — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Additionally, if you have elevated [permissions](permissions) to General, there'
 * [triage] You can close PRs if the package author requests it or the registration is superseded by another registration request.
 * [write] You can merge PRs that have the _needs to be manually merged in 3 days_ label once the requisite waiting period has passed, assuming there are no outstanding objections in the PR comments.
 * [write] You can choose to facilitate expedited merge requests, after manually reviewing the package.
-You generally should not merge your own registrations (though you can make requests to another maintainer).
+You generally should not merge your own registrations (though you can make requests to another maintainer) as mentioned in the [README](https://github.com/JuliaRegistries/General/#who-can-approve-an-early-merge)
```
Comment (ericphanson) [inline CONTRIBUTING.md:89]:
```suggestion
You generally should not merge your own registrations or those you are involved with (though you can make requests to another maintainer). See also [this FAQ entry](https://github.com/JuliaRegistries/General/#who-can-approve-an-early-merge).
```

File: README.md [hunk 2]
```diff
@@ -254,6 +254,19 @@ See [these](https://github.com/JuliaRegistries/General/pull/31549#issuecomment-7
 [^1]: Note that even within the world of OSI licenses, there are combinations of OSI licenses which are not
 legal to use together, such as GPL2 with Apache2.
 
+#### Can my package registration PR be merged early?
+
+New packages must typically wait for the 3 day timer to expire before they are merged, either automatically by the bot or manually. 
+However, exceptions may be made with proper justification, and opportunity for public comment, for instance if there is a time constraint. 
+To make this request please post either in the [#pkg-registration](https://julialang.slack.com/archives/C6M4DQA5P) channel on the 
+public Julia Language Slack, or directly on the PR itself. Proper justification should be provided on the pull request.
```
Comment (ericphanson) [inline README.md:262]: I think we should try to be more specific than "proper justification":
```suggestion
This waiting period allows time for community comment and review, and to provide time to check things like:

- the name is appropriate (e.g. "safe for work", not typosquatting, and fitting with the [Pkg naming guidelines](https://pkgdocs.julialang.org/v1/creating-packages/#Package-naming-guidelines))
- the package is functional and non-empty
- the code is not malicious (e.g. exploiting security vulnerabilities)
- there is some form of README or documentation to enable users to use the package

If there is some urgent need for the package to be registered (for example, an upcoming talk featuring the package or other external deadline), the waiting period can be expedited by request made in the [#pkg-registration](https://julialang.slack.com/archives/C6M4DQA5P) channel on the 
public Julia Language Slack, or directly on the PR itself. This request should include a link to the registration PR and why an expedited merge is needed. In this case, a registry maintainer (see below) can manually review the package (often with stricter standards than the typical package faces, given the lack of bandwidth to review all packages thoroughly).
```

**Review by ericphanson — APPROVED** [review]

**Review by vchuravy — APPROVED** [review]

**Comment by ericphanson** [comment]
I'll merge this now since it seems to have some consensus, but if there is post-merge feedback we can always make a followup PR.

**Pull request merged**